routes/eth.go

```diff
@@ -422,3 +529,14 @@ func (fes *APIServer) GetETHTransactionByHash(hash string) (_tx *InfuraTx, _err
 	}
 	return response, nil
 }
+func publicKeyToEthAddress(address []byte) (str string, err error) {
+	addressPubKey, err := btcutil.NewAddressPubKey(address, &chaincfg.MainNetParams)
+	if err != nil {
+		panic(err)
```

Comment:
Please just return the error.
```suggestion
		return "", err
```